config/webpack.dev.config.js

```diff
@@ -87,7 +98,6 @@ module.exports = Merge.smart(commonConfig, {
       filename: 'accessibilityPolicy.html',
       template: path.resolve(__dirname, '../public/index.html'),
     }),
-    new webpack.HotModuleReplacementPlugin(),
```

Comment:
why is this being taken out?

Reply:
I think I remember webpack complaining about this. Because [our npm script "start"](https://github.com/edx/studio-frontend/blob/master/package.json#L10) has `--hot` it already automatically adds this plugin. So, this line is redundant.